core/src/main/java/org/apache/accumulo/core/data/LoadPlan.java

```diff
@@ -162,6 +162,33 @@ public byte[] getEndRow() {
     public RangeType getRangeType() {
       return rangeType;
     }
+
+    @Override
+    public int hashCode() {
+      final int prime = 31;
+      int result = 1;
+      result = prime * result + Arrays.hashCode(endRow);
+      result = prime * result + Arrays.hashCode(startRow);
+      result = prime * result + Objects.hash(fileName, rangeType);
+      return result;
```

Comment:
Could simplify this to

```suggestion
     return Objects.hash(Arrays.hashCode(endRow), Arrays.hashCode(startRow), fileName, rangeType);
```